modules/mono/editor/GodotTools/GodotTools.IdeMessaging/Utils/NotifyAwaiter.cs

```diff
@@ -1,17 +1,20 @@
+// ReSharper disable ParameterHidesMember
+// ReSharper disable UnusedMember.Global
+
 using System;
 using System.Runtime.CompilerServices;
 
 namespace GodotTools.IdeMessaging.Utils
 {
     public class NotifyAwaiter<T> : INotifyCompletion
     {
-        private Action continuation;
-        private Exception exception;
-        private T result;
+        private Action? continuation;
+        private Exception? exception;
+        private T? result;
 
         public bool IsCompleted { get; private set; }
 
-        public T GetResult()
+        public T? GetResult()
```

Comment:
This should probably not be nullable.

```suggestion
        public T GetResult()
```